libclc/generic/include/clc/float/definitions.h

```diff
@@ -31,6 +31,8 @@
 #define M_SQRT2_F       0x1.6a09e6p+0f
 #define M_SQRT1_2_F     0x1.6a09e6p-1f
 
+#define M_PI_OVER_180_F 0x1.1df46ap-6F
```

Comment:
```suggestion
#define M_PI_OVER_180_F 0x1.1df46ap-6f
```

What are new macro for? I think it might be better introduce them in the PRs where they used to simplify review and "cherry-picking".

This particular change is not relevant to enabling half for CUDA target.

Reply:
That's a legacy of trying to understand what this value were.

But you are right, they don't relate the patch. I reverted them.

Reply:
`libclc/generic/include/clc/float/definitions.h` changes look okay to me, although I don't know how to verify the values. 
I found uses of added defines in libclc.